README.md

```diff
@@ -555,6 +555,41 @@ Here are the metrics settings:
     "vm.max_map_count" = "262144"
     ```
 
+#### Boot-related settings
+
+*Please note that boot settings only exist for bare-metal variants at the moment*
+
+Specifying either of the following settings will generate a kernel boot config file to be loaded on subsequent boots:
+* `settings.boot.kernel-parameters`: These allows additional kernel parameters to be specified to the kernel command line during boot.
+* `settings.boot.init-parameters`: These allows additional init parameters to be specified to init during boot.
+
+Example user data for specifying boot settings:
+
+```toml
+[settings.boot.kernel]
```

Comment:
Both are acceptable since I added `kernel` as an alias for `kernel-parameters` to serde. You're right that it should probably be consistent in the documentation.